### PR TITLE
update occurrences of starter kit

### PR DESCRIPTION
**Pull request by harishv7**
### TL;DR

Renamed "Starter Kit" to "Isomer Studio" across the project.

### What changed?

- Updated the app name from "Starter Kit" to "Isomer Studio" in various files:
  - DevContainer configuration
  - Storybook preview
  - README.md
  - Environment variables
  - Landing page header
- Added a reference to Starter Kit in the README.md file

### How to test?

1. Check the DevContainer name in `.devcontainer/devcontainer.json`
2. Run Storybook and verify the app name in the preview
3. Review the README.md file for updated content
4. Verify the default app name in `src/env.mjs`
5. Run the application and check the landing page header

### Why make this change?

This change establishes "Isomer Studio" as the distinct identity for the project while acknowledging its foundation in the Starter Kit. It ensures consistency across the codebase and user-facing elements, providing a clear brand for the application.

### CODE REVIEW

**Comment by harishv7** [comment]
* **#596** <a href="https://app.graphite.dev/github/pr/opengovsg/isomer/596?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @harishv7 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/studio/.storybook/preview.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ const trpc = createTRPCReact<AppRouter>()
 const StorybookEnvDecorator: Decorator = (story) => {
   const mockEnv: EnvContextReturn["env"] = merge(
     {
-      NEXT_PUBLIC_APP_NAME: "Starter Kit",
+      NEXT_PUBLIC_APP_NAME: " Isomer Studio",
```
Comment (seaerchin) [inline apps/studio/.storybook/preview.tsx:46]: nit: trim the starting space please!

**Review by seaerchin — APPROVED** [review]
lgtm. checked via incognito - saw the login page updated and the new tab also

**Comment by harishv7** [comment]
### Merge activity

* **Sep 11, 3:18 AM EDT**: @harishv7 merged this pull request with [Graphite](https://app.graphite.dev/github/pr/opengovsg/isomer/596).

**Pull request merged**